src/webapp/public/locales/en/translation.json

```diff
@@ -23,7 +23,14 @@
           "timer_fade_volume": "Fade volume",
           "toggle_output": "Toggle audio output",
           "sync_rfidcards_all": "Sync all audiofiles and card entries",
-          "sync_rfidcards_change_on_rfid_scan": "Change activation of 'on RFID scan'"
+          "sync_rfidcards_change_on_rfid_scan": "Change activation of 'on RFID scan'",
+          "next_song": "Next song",
+          "pause": "Pause",
+          "play": "Play",
+          "prev_song": "Previous song",
+          "shuffle": "Toggle Shuffle",
```

Comment:
"De-/Activate Shuffle" or "Activate/Deactivate Shuffle", like the description for shuffle in line 147?

Reply:
"Toggle" is a typical word to describe what is supposed to do. It's just not as descriptive in German, for example. It's also better in the UI when descriptions are shorter (e.g. for screen reader users).